plezanje-net/web
# Edit activity / Add routes to activity

BE PR to test saving the forms with https://github.com/plezanje-net/api/pull/98

testing scenarios: 
- edit existing ativity from the menu, edit some data that is not disabled
- check a few routes and select a date that already has a log entry for the date - it should change the form to adding / editing mode/

Changed around conditioning of certain fields a bit and made data passing more transparent (without sending in additional type input).

Returned icefall and peak options but with a hint that you shouldn't log like this if the entity exists in our DB.
